docs/setup/installation/platforms/docker.md

```diff
@@ -25,10 +25,6 @@ docker run -v $PWD/vector.toml:/etc/vector/vector.toml:ro timberio/vector-slim:l
 ```
 {% endcode-tabs-item %}
 {% code-tabs-item title="alpine" %}
-{% hint style="warning" %}
```

Comment:
is there a reason this change is included in this PR?

Reply:
Yeah, it removes the warning that this PR fixes

Reply:
👍